EIPS/eip-3651.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 3651
+title: Warm COINBASE
+author: William Morriss (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-3651-warm-coinbase/6640
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-07-12
+requires: 2929
+---
+
+## Simple Summary
+Starts the `COINBASE` address warm
+
+## Abstract
+The `COINBASE` address shall be warm at the start of transaction execution, in accordance with the actual cost of reading that account.
+
+## Motivation
+Direct `COINBASE` payments are becoming increasingly popular because they allow conditional payments, which provide benefits such as implicit cancellation of transactions that would revert.
```

Comment:
@wjmelements 
Hi, could you explain what is `Direct COINBASE payments` ? thanks in advance.
BTW, the EIP still encourage users to paying for transactions on Ethereum?